mautic/mautic
# Fixed binding modal events multiple times

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      |
| Related developer documentation PR URL |
| Issue(s) addressed                     |



#### Description:
I found out that there is a JS code that binds event listeners multiple times to `show.bs.modal` and `hidden.bs.modal` events. This PR fixes this issue

#### Steps to reproduce:
1. Place breakpoint on `app/bundles/CoreBundle/Assets/js/9.modals.js` line 68.
2. Go to `/index_dev.php/s/dashboard`.
3. Click the `Plugins` menu item in the right menu.
4. You should reach the breakpoint twice.

#### Steps to test this PR:
1. Place breakpoint on `app/bundles/CoreBundle/Assets/js/9.modals.js` line 68.
2. Go to `/index_dev.php/s/dashboard`.
3. Click the `Plugins` menu item in the right menu.
4. You should reach the breakpoint only once.
